doc/development/rfc/README.md

```diff
@@ -0,0 +1,26 @@
+# Request For Comment (RFC) List
+
+GRASS Project Steering Committee:
+
+- [GRASS-PSC](https://grasswiki.osgeo.org/wiki/PSC)
+
+A list of all GRASS GIS RFC documents, with status:
+
+- [RFC 1: Project Steering Committee Guidelines](PSC_guidelines.md) (Adopted)
+- [RFC 2: Legal aspects of code contributions](legal_aspects_of_code_contributions.md)
+  (Adopted)
+- [RFC 3: PSC Voting Procedures](PSC_voting_procedures.md) (Adopted)
+- [RFC 4: Release Procedure](https://trac.osgeo.org/grass/wiki/RFC/4_ReleaseProcedure)
```

Comment:
Should these point to the PRs, not trac?
https://github.com/OSGeo/grass/pulls?q=is%3Apr+is%3Aopen+label%3ARFC+

Reply:
Sure, changed.